Parcare/Models/ParkingLotModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Parking.Models
+{
+    public class ParkingLotModel
+    {
+        public required string CarNumber { get; set; }
+        public required DateTime EntryTime { get; set; }
+        public TimeSpan Duration { get; set; }
```

Comment:
Do you need duration here? Maybe something like isCarOut..

Reply:
Not really